about.html

```diff
@@ -12,62 +13,100 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+  <div class="container about-page">
+    <header>
+      <img class="home-top-img" src="img/lambda-black.png" alt="Header Img">
```

Comment:
one small tip! you can wrap the logo in an a tag and have it point to the home page!